entrypoint.js

```diff
@@ -0,0 +1,29 @@
+const core = require("@actions/core");
```

Comment:
I'd rather not rename this entrypoint, and stick with index.js to match the docs and the [example action](https://github.com/actions/javascript-action) as following that pattern is probably going to match other actions out there too.

I'd also like to not have all the functions in `index.js`, and rather have separate files with the function names to split them up, but that can be done later.

Leave a comment during code review if you'd like to make those changes, or I'll just leave it for now.